### ISSUE TITLE
empirical mean from tf.random.gamma does not match theoretical expectation

### ISSUE THREAD

**Issue by corcra** [tensorflow/tensorflow#33476]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Linux Ubuntu 16.04.6
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
N/A
- TensorFlow installed from (source or binary):
Not sure, came pre-installed so probably binary
- TensorFlow version (use command below):
1.13.1
- Python version:
3.5.5
- Bazel version (if compiling from source):
N/A
- GCC/Compiler version (if compiling from source):
N/A
- CUDA/cuDNN version:
CUDA: 10.0.130
cuDNN: 7
- GPU model and memory:
Tesla K80, total memory = 11441 MiB

**Describe the current behavior**
If I sample many values from `tf.random.gamma` with shape=A and beta=B and compute their mean, I get a value which isn't close to A/B.

If I use the script below, computing the mean absolute difference between the empirical and theoretical mean, I get ~7.26 (with 500 tries, 50k samples from the gamma distribution each time).

**Describe the expected behavior**
The expected value of a Gamma-distributed random variable with shape parameter A and inverse scale parameter B is A/B, so the empirical mean should be close to A/B. 

So I would expect the output of my script to be close to 0.

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

Code to report the mean absolute discrepancy between the empirical and theoretical means:
```
def check_gamma(n_sample=50000, n_iter=500):
    shape_ph = tf.placeholder(tf.float32)
    beta_ph = tf.placeholder(tf.float32)
    gamma_mean = tf.reduce_mean(tf.random.gamma(shape=[n_sample], alpha=shape_ph, beta=beta_ph))
    mean_difference = np.zeros(shape=n_iter)
    mean_difference_np = np.zeros(shape=n_iter)
    with tf.Session() as sess:
        for i in range(n_iter):
            # sample a shape (non-negative)
            shape_parameter = np.random.uniform(1, 5)
            # sample a scale (non-negative)
            scale_parameter = np.random.uniform(1, 5)
            # the gamma distribution in tensorflow takes the inverse scale parameter
            beta = 1.0/scale_parameter
            # theoretical mean of gamma distribution:
            expected_mean = shape_parameter*scale_parameter
            # sample it!
            sample_mean = sess.run(gamma_mean, feed_dict={shape_ph: shape_parameter, beta_ph: beta})
            mean_difference[i] = np.abs(expected_mean - sample_mean)
            mean_difference_np[i] = np.abs(expected_mean - np.mean(np.random.gamma(size=n_sample, shape=shape_parameter, scale=scale_parameter)))
    print('average absolute difference in empirical and theoretical means, using tensorflow:', np.mean(mean_difference))
    print('...using numpy:', np.mean(mean_difference_np))
    return True
```

Code to visualise the distribution:
```def visualise_distribution(n_sample=10000, shape_parameter=1, scale_parameter=2):
    with tf.Session() as sess:
        gamma_samples = sess.run(tf.random.gamma(shape=[n_sample], alpha=shape_parameter, beta=1.0/scale_parameter))
    expected_mean = shape_parameter * scale_parameter
    empirical_mean = np.mean(gamma_samples)
    sns.distplot(gamma_samples, kde=False)
    plt.axvline(x=expected_mean, label='theoretical mean', color='black')
    plt.axvline(x=empirical_mean, label='empirical mean')
    plt.legend()
    return True
```

**Other info / logs**

Visualisation of distribution produced by above code, with shape = 1, scale = 2:
![image](https://user-images.githubusercontent.com/4471845/67024197-ea4ec880-f0fb-11e9-809d-8b9f338598cf.png)

**Comment by oanush** [comment]
@corcra ,
Thank you for reporting, can you please provide a standalone code so that we can try reproducing the issue.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by oanush** [comment]
@corcra ,
Any update on the issue ?Thanks!

**Comment by oanush** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
